app/screens/wallet/Backup.js

```diff
@@ -0,0 +1,187 @@
+// @flow
+import React, { Component } from 'react';
+import styled, { css } from 'styled-components';
+import { smColors } from '/vars';
+import { shieldIconGreenOne, shieldIconOrangeTwo } from '/assets/images';
+
+const Actionable = css`
```

Comment:
this component is used only once

Reply:
integrated with ActionLink styled component